### PR TITLE
[master] Adding clarity to CPU Architecture compatibility

### PR DESCRIPTION
**Pull request by sanjulamadurapperuma**
## Purpose
Fixes [wso2/product-is#12903](https://github.com/wso2/product-is/issues/12903)

## Documentation
[1] - [https://is.docs.wso2.com/en/latest/setup/installation-prerequisites/](https://is.docs.wso2.com/en/latest/setup/installation-prerequisites/)
[2] - [https://is.docs.wso2.com/en/5.11.0/setup/deployment-guide/](https://is.docs.wso2.com/en/5.11.0/setup/deployment-guide/)

### CODE REVIEW

File: en/docs/setup/deployment-guide.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ necessary system requirements and a compatible environment.
 </tbody>
 </table>
 !!! note
-    For more information on prerequisites, see [Installation Prerequisites](../../setup/installation-prerequisites)
+    We do not recommend running WSO2 Identity Server on ARM architecture. However, official support for this is being considered for future releases.
+    
+    For more information on prerequisites, see [Installation Prerequisites](../../setup/installation-prerequisites).
```
Comment (nilminiwso2) [inline en/docs/setup/deployment-guide.md:47]:
```suggestion
    For more information see the [installation prerequisites](../../setup/installation-prerequisites).
```

File: en/docs/setup/deployment-guide.md [hunk 2]
```diff
@@ -42,7 +42,9 @@ necessary system requirements and a compatible environment.
 </tbody>
 </table>
 !!! note
-    For more information on prerequisites, see [Installation Prerequisites](../../setup/installation-prerequisites)
+    We do not recommend running WSO2 Identity Server on ARM architecture. However, official support for this is being considered for future releases.
```
Comment (nilminiwso2) [inline en/docs/setup/deployment-guide.md:45]: Is this `Arm architecture`?
@sanjulamadurapperuma

Comment (sanjulamadurapperuma) [reply en/docs/setup/deployment-guide.md:45]: Yes, that's correct

**Comment by sanjulamadurapperuma** [comment]
@nilminiwso2 The requested changes have been made. Thanks for your input!

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**